README.md

```diff
@@ -0,0 +1,713 @@
+    },
+    "es": {
+        "metadata": {...},
+        "data": {
+            "ñ": {...}
+        }
+    }
+}
+*/
+```
+
+If no diacritics are found in the string, an object with an error "message" is returned.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var test = diacritics.getDiacritics("test");
+// test => { "message": "No diacritics found" }
+```
+
+### diacritics.getBase(array)
+
+Returns the base data for the matching base value within the array.
+
+The `array` parameter contains either an `array` of diacritic base string, or a `string` containing a single base (internally converted into an array), of characters to process.
+
+The returned value will be an object containing all languages that match each diacritic base value, but the language data will only contain the matching diacritic.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var data = diacritics.getBase(["u"]);
+/* data =>
+{
+    "de": {
+        "metadata": {...},
+        "data: {
+            "ü": {...}
+        }
+    },
+    "es": {
+        "metadata": {...},
+        "data": {
+            "ú": {...},
+            "ü": {...}
+        }
+    }
+}
+*/
+```
+
+If no matching bases are found in the database, an object with an error "message" is returned.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var test = diacritics.getBase(["&"]);
+// test => { "message": "No matching bases found" }
+```
+
+### diacritics.getDecompose(array)
+
+Returns the decompose data for the matching decompose values within the array.
+
+The `array` parameter contains either an `array` of diacritic decompose string, or a `string` containing a single decompose string (internally converted into an array), of characters to process.
+
+The returned value will be an object containing all languages that match each diacritic decompose value, but the language data will only contain the matching diacritic.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var data = diacritics.getDecompose(["ss"]);
+/* data =>
+{
+    "de": {
+        "metadata": {...},
+        "data: {
+            "ß": {...}
+        }
+    }
+}
+*/
+```
+
+If no matching decomposes are found in the database, an object with an error "message" is returned.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var test = diacritics.getDecompose(["&", "test"]);
+// test => { "message": "No matching decomposes found" }
+```
+
+## Transliteration functions
+
+### diacritics.transliterate(string [, type][, variant])
+
+Replaces diacritics within the string with either the base or decomposed value.
+
+- The `string` parameter (type: `string`) will replace any diacritic characters with base or decomposed values. Any non-diacritics characters will not be modified.
+- The `type` parameter (type: `string`) is set to either "base" or "decompose". Defaults to "base". If a "base" value is not found, a "decompose" value will be used instead; and vice-versa.
```

Comment:
I don't understand this sentence: `If a "base" value is not found, a "decompose" value will be used instead; and vice-versa.`. So, what's the fallback, `base` or `decompose`?